web_timeline/readme/CONFIGURE.rst

```diff
@@ -0,0 +1,99 @@
++--------------------+-----------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| stack              | No        | When set to false, items will not be stacked on top of each other such that they do overlap.                                                                                                                                                                              |
++--------------------+-----------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| colors             | No        | Allows to set certain specific colors if the expressed condition (JS syntax) is met.                                                                                                                                                                                      |
++--------------------+-----------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| dependency_arrow   | No        | Set this attribute to a x2many field to draw arrows between the records referenced in the x2many field.                                                                                                                                                                   |
++--------------------+-----------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+Optionally you can declare a custom template, which will be used to render the
+timeline items. You have to name the template 'timeline-item'.
+These are the variables available in template rendering:
+
+* ``record``: to access the fields values selected in the timeline definition.
+* ``field_utils``: used to format and parse values (see available functions in ``web.field_utils``).
+
+You also need to declare the view in an action window of the involved model.
+
+Example:
+
+.. code-block:: xml
+
+    <?xml version="1.0" encoding="utf-8"?>
+    <odoo>
+        <record id="view_task_timeline" model="ir.ui.view">
+            <field name="model">project.task</field>
+            <field name="type">timeline</field>
+            <field name="arch" type="xml">
+                <timeline date_start="date_start"
```

Comment:
We should change field to `date_assign` as we do not have field `date_start` in model `project.task`